components/ibm-components/ffdl/train/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6-slim
+
+RUN pip install boto3 ruamel.yaml requests
+
+ENV APP_HOME /app
+COPY src $APP_HOME
+WORKDIR $APP_HOME
+
+ENTRYPOINT ["python", "train.py"]
```

Comment:
Could you document what the components/ibm-components/ffdl/train/src/ffdl binary does (ideally in a README.md)? Is there any licensing information that should be associated with this binary?

Reply:
@vicaire the binary is FfDL CLI. Its Apache 2.0, as is the whole code. We will add a line vis a vis in README